py/tests/rest/htm_rest_api_test.py

```diff
@@ -200,7 +200,7 @@ def tearDown(self):
       r = requests.get('{}/stop'.format(HOST))
       print(r)
     except:
-      pass
+      os.system("ps x | grep rest_server | grep -v grep | awk '{print $1}' | xargs kill -9")
```

Comment:
I guess this will only run on Unixes, not Windows. On the other hand, I wouldn't worry too much about stopping the server? Since the intended use is that the user runs manually the server (so they should as well bring it down). Ofc this is a problem for the automated test - wouldn't the OS kill all threads spawned from the (unit-test) process once the parent finishes?

Reply:
I' ll remove it.

Reply:
> wouldn't the OS kill all threads spawned from the (unit-test) process once the parent finishes?

Good point.  For the unit test we could just remove the line `threadObj.join()`.  
If the server does not process the /stop then it will be killed when the test program finishes.  No problem with unknown state or anything.  Everything is clean.

In a general case where the server is in a different machine we would not want to be shutting down the server.  So this is good.

Thanks.